concepticondata/conceptlists/Hale-1973-1798.tsv

```diff
@@ -272,7 +272,7 @@ Hale-1973-1798-270	Human_Body-171	cough	v	02b2.26	879	COUGH
 Hale-1973-1798-271	Human_Body-172	sneeze	v	02b2.27	1621	SNEEZE
 Hale-1973-1798-272	Human_Body-173	itch	v	02b2.28	3855	ITCH (CAUSE ITCHING OR FEEL ITCHY)
 Hale-1973-1798-273	Human_Body-174	vomit	v	02b2.29	1278	VOMIT
-Hale-1973-1798-274	Human_Body-175	be numb	v	02b2.30		
+Hale-1973-1798-274	Human_Body-175	be numb	v	02b2.30	2216	BECOME NUMB
```

Comment:
[Bodt-2019-664-583](https://concepticon.clld.org/values/Bodt-2019-664-583) also has "be numb" and there it's unmapped, so would leave it unmapped here too.

Reply:
I would map it based on the part-of-speech tag "v".